packages/http/src/models/Transaction.ts

```diff
@@ -495,7 +494,11 @@ function prepareTxn(txn: any, { deep } = { deep: false }) {
 
   balanceConversions.forEach(({ key, currencyType }) => {
     if (txn[key] && typeof txn[key] === 'number') {
-      txn[key] = new Balance(txn[key], currencyType)
+      if (txn.token_type !== undefined) {
```

Comment:
also wondering if we should handle testnet here too

Reply:
I'm not sure if `token_type` for testnet will be `'tnt'` or `'hnt'`, but that would affect how we handle testnet